jobs/deploy-buildmaster.groovy

```diff
@@ -28,20 +22,20 @@ new Setup(steps
     ["khan-test", "khan-internal-services"],
 ).apply();
 
-currentBuild.displayName = "${currentBuild.displayName} (${params.SERVICE} - ${params.BRANCH})";
+currentBuild.displayName = "${currentBuild.displayName} - ${params.GIT_BRANCH})";
 
 def installDeps() {
-   kaGit.safeSyncToOrigin("git@github.com:Khan/buildmaster2", "${params.BRANCH}");
+   kaGit.quickClone("git@github.com:Khan/buildmaster2", "buildmaster2", params.GIT_BRANCH);
```

Comment:
Eventually, i think we can use the `buildmaster` repo instead.  We can rename https://github.com/Khan/buildmaster to https://github.com/Khan/buildmaster-deprecated and rename https://github.com/Khan/buildmaster2 to https://github.com/Khan/buildmaster.

Future improvement tho.